src/main/target/COLIBRI_RACE/i2c_bst.c

```diff
@@ -372,9 +372,9 @@ static bool bstSlaveProcessFeedbackCommand(uint8_t bstRequest)
             bstWrite16(compassConfig()->mag_declination / 10);
 
             bstWrite8(voltageSensorADCConfig(VOLTAGE_SENSOR_ADC_VBAT)->vbatscale);
-            bstWrite8(batteryConfig()->vbatmincellvoltage);
-            bstWrite8(batteryConfig()->vbatmaxcellvoltage);
-            bstWrite8(batteryConfig()->vbatwarningcellvoltage);
+            bstWrite8(batteryConfig()->(vbatmincellvoltage + 5) / 10);
```

Comment:
Brace in the wrong place.

Reply:
Yeah, I found out, fixed but couldn't commit